manifests/0000_20_kube-apiserver-operator_07_clusteroperator.yaml

```diff
@@ -39,3 +39,6 @@ status:
       resource: mutatingwebhookconfigurations
     - group: admissionregistration.k8s.io
       resource: validatingwebhookconfigurations
+    - group: controlplane.operator.openshift.io
```

Comment:
you also need to add this to the bit in starter.go that duplicates this information.  This is used before the operator starts, the other is used afterwards.

Reply:
Done.

Reply:
@sanchezl can you remind me that I want to fix up `oc admin inspect` to put all the instances in a single file?

Reply:
well, I do and I don't :(  I guess I want a visualizer for it.  ugh.